### ISSUE TITLE
Long text field truncated via odbc()

### ISSUE THREAD

**Issue by AppKiv** [ClickHouse/ClickHouse#9363]
ODBC-bridge to postgres, truncate long string.
ver 19.17.6.36

1. postgres:
```
drop table if exists tmp.longtext;
create table tmp.longtext(flen int, field1 text);
tmp.longtext(flen,field1)values(3248,'BEGIN long text, see attachment... ')
```
[postgres.sql.txt](https://github.com/ClickHouse/ClickHouse/files/4250080/postgres.sql.txt)

2. clickhouse
```
select LENGTH(field1), flen, field1 from odbc('DSN=DWH;','tmp','longtext')
```

Data in "filed1" would be truncated to 1024.
I've try set "odbc_max_field_size" but it's not help, also I've investigate code and have not found where this setting will be used.
I've read data in python script via odbc driver - it's work correct, so I think odbc-bdridge truncate long string.

**Comment by waSRull** [comment]
have same trouble

**Comment by jathael** [comment]
Below added to odbc.ini worked for me:

`MaxCharSize         = 2000000
MaxVarcharSize      = 2000000
MaxLongVarcharSize  = 2000000`

**Comment by lazjake** [comment]
i'm running into the same issue, except returning long text fields truncated to 1023 chars. and no luck adding `Max...Size` options to odbc.ini. on clickhouse version 20.2.12.112. any luck figuring this one out?

**Comment by alexey-milovidov** [comment]
The fix is going to be released in version 21.5.

**Comment by alexey-milovidov** [comment]
21.5-testing is already available in repositories.
